pcsx2/PAD/Linux/SDL/joystick.cpp

```diff
@@ -259,14 +270,22 @@ size_t JoystickInfo::GetUniqueIdentifier()
 
 bool JoystickInfo::TestForce(float strength = 0.60)
 {
+	static const Uint32 test_duration_ms = 400;
+	bool working;
```

Comment:
I'd name the variable to something else, maybe rumble_enabled ?

Reply:
Fixed